.github/workflows/ci.yml

```diff
@@ -112,3 +116,31 @@ jobs:
 
             If this is not what was expected, commit changes to `config/versions.json`.
         run: gh pr comment --body '${{ env.BODY }}'
+
+  prerelease-deploy-version:
+    name: Get Prerelease Number
+    runs-on: ubuntu-latest
```

Comment:
This doesn't require explicit `needs` because the workflow will just fail if either of the previous workflows fail?